File: .ProgramingBasics/.SimpleCalculations/06.CircleAreaAndPerimeter/src/CircleAreaAndPerimeter.java
import java.util.Scanner;

public class CircleAreaAndPerimeter {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        double r = scanner.nextDouble();
        double area = Math.PI * Math.pow(r, 2);
        double perimeter = 2 * Math.PI * r;

        System.out.println("Area = " + area);
        System.out.println("Perimeter = " + perimeter);
    }
}

File: ProgramingBasics/SimpleCalculations/06.CircleAreaAndPerimeter/06.CircleAreaAndPerimeter/CircleAreaAndPerimeter.cs
using System;

namespace _06.CircleAreaAndPerimeter
{
    public class CircleAreaAndPerimeter
    {
        public static void Main(string[] args)
        {
            var r = double.Parse(Console.ReadLine());
            var area = Math.PI * r * r;
            var perimeter = 2 * Math.PI * r;

            Console.WriteLine($"Area = {area}");
            Console.WriteLine($"Perimeter = {perimeter}");
        }
    }
}
